cmd/nvidia-dra-plugin/sharing.go

```diff
@@ -101,17 +101,32 @@ func (t *TimeSlicingManager) SetTimeSlice(devices *PreparedDevices, config *nasc
 		return fmt.Errorf("setting a TimeSlice duration on MIG devices is unsupported")
 	}
 
+	noSupportTimeSliceIDs := []string{}
+	for _, gpu := range devices.Gpu.Devices {
+		if !detectSupportTimeSliceByArch(gpu.architecture) {
+			klog.InfoS("the current card does not support setting time slices and will be ignored.", "arch", gpu.architecture, "uuid", gpu.uuid, "cudaComputeCapability", gpu.cudaComputeCapability)
+			noSupportTimeSliceIDs = append(noSupportTimeSliceIDs, gpu.uuid)
+			continue
+		}
+	}
```

Comment:
```suggestion
	var supportedDeviceUUIDs []string
	for _, gpu := range devices.Gpu.Devices {
		if detectSupportTimeSliceByArch(gpu.architecture) {
			supportedDeviceUUIDs = append(supportedDeviceUUIDs, gpu.uuid)
			continue
		}
		klog.InfoS("the current card does not support setting time slices and will be ignored.", "arch", gpu.architecture, "uuid", gpu.uuid, "cudaComputeCapability", gpu.cudaComputeCapability)
	}
```

Reply:
This way the code will be much clearer, thanks for the review